pkg/timelock/scheduler.go

```diff
@@ -113,40 +126,38 @@ func (tw *Worker) updateSchedulerDelay(t time.Duration) {
 }
 
 // addToScheduler adds a new CallSchedule operation safely to the store.
-func (tw *Worker) addToScheduler(op *contract.TimelockCallScheduled) {
+func (tw *scheduler) addToScheduler(op *contract.TimelockCallScheduled) {
 	tw.logger.Debug().Msgf("scheduling operation: %x", op.Id)
 	tw.add <- op
-	tw.logger.Debug().Msgf("operations in scheduler: %v", len(tw.store))
```

Comment:
This `len(tw.store)` call is causing a deadlock, so I'm removing it from `addToScheduler` and `delFromScheduler`.

The alternative would be to refactor the scheduler implementation, which I've started doing in [this branch](https://github.com/smartcontractkit/timelock-worker/tree/feat-refactor-scheduler). It seems to work as expected but it's a risky change which I'd rather not worry about right now. Once we've enhanced our test suite I'll give it another go.